pandas/tests/indexes/common.py

```diff
@@ -329,6 +329,25 @@ def test_duplicates(self, indices):
         assert not idx.is_unique
         assert idx.has_duplicates
 
+    def test_unique(self):
+        # GH 17896
```

Comment:
really like to use the indices fixture here to test all Index types

Reply:
It assigns no name to the index... shall I change it so it does?

Reply:
you can try that